File: 프로그래머스/1/132267. 콜라 문제/콜라 문제.py
def solution(a, b, n):
    # 빈병 a개를 콜라 b개로 바꿔줌 n개면?
    answer = 0
    while 1:
        if n // a == 0:
            break
        newBottle = n // a
        newBottle *= b
        emptyBottle = n % a
        n = emptyBottle + newBottle
        answer += newBottle
    
    return answer

File: 프로그래머스/1/132267. 콜라 문제/콜라 문제.java
class Solution {
    public int solution(int a, int b, int n) {
        int answer = 0;
        while(true){
            if(n / a < 1) break;
            int temp = (n / a) * b;
            answer += temp;
            n = temp + (n % a);
        }
        return answer;
    }
}